libs/internal-dns/chainsaw-test/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: install-internal-dns
```

Comment:
Remove all the namespace definitions as chainsaw adds a namespace itself. When instantiating helm chart use the $namespace variable so the helm chart is created in the right place

Reply:
Updated to use $namespace variable in commit 30d2916